src/Init/Data/Int/Basic.lean

```diff
@@ -333,6 +333,13 @@ instance : Min Int := minOfLe
 
 instance : Max Int := maxOfLe
 
+/--
+Return the absolute value of an integer.
+-/
+def abs : Int → Int
+  | ofNat n   => .ofNat n
+  | negSucc n => .ofNat n.succ
```

Comment:
Is this not just [`Int.natAbs`](https://leanprover-community.github.io/mathlib4_docs/Init/Data/Int/Basic.html#Int.natAbs) plus a coercion from Nat to Int?

Reply:
Certainly. Is that how you'd like me to spell it?